lib/utils/constants.dart

```diff
@@ -91,7 +90,7 @@ const int kNumberOfSeconds = 1000;
 const int kSixtySeconds = 60;
 const int kFileCompressQuality = 50;
 
-const double kPrecision = 100000000000000000;
+const double kPrecision = 10000000000000000;
```

Comment:
Can you check before changing is it used somewhere and this change doesnt effect any.